tests/module_api/test_api.py

```diff
@@ -635,6 +636,34 @@ def test_check_push_rules_actions(self) -> None:
             [{"set_tweak": "sound", "value": "default"}]
         )
 
+    def test_create_room(self) -> None:
```

Comment:
I think it would be a good thing to also check that we get the right alias back when we create the room with an alias.

Reply:
Added here: bbc85b6